src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import { Checkbox, Form, FormGroup, Select, SelectOption, Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+import { DownloadIcon } from '@patternfly/react-icons';
+import routerParams from '../../../../Utilities/RouterParams';
+import { SimpleTableFilter } from '../../../../PresentationalComponents/SimpleTableFilter';
+import debounce from 'lodash/debounce';
+import propTypes from 'prop-types';
+
+const CVSSOptions = [
+    { value: 'all', label: 'All', disabled: false, from: '', to: '' },
+    { value: '0to3', label: 'Lower than 3', disabled: false, from: '', to: 3 },
+    { value: '3to7', label: '3 - 7', disabled: false, from: 3, to: 7 },
+    { value: '7to10', label: '7 - 10', disabled: false, from: 7, to: 10 }
+];
+
+class VulnerabilitiesCveTableToolbar extends Component {
+    state = {
+        defaultConfig: {
+            filter: '',
+            // eslint-disable-next-line camelcase
```

Comment:
if you're doing this in two places, time to hoist it to the top, do it globally for the file 🤔 (maybe? )

Reply:
I would prefer not use these disables for camelcase, but well that is task for someone in vuln team or later timing. However gonna turn off that rule for whole file